packages/optimistic-oracle/src/proposer.js

```diff
@@ -195,7 +193,6 @@ class OptimisticOracleProposer {
         priceRequest,
         proposalBond: returnValue,
         proposalPrice,
-        transactionConfig,
```

Comment:
Removed from logs because `transactionConfig` is missing a `gas` param (which we purposefully omit to allow Truffle to estimate gas) which causes issues for SlackTransport. SlackTransport sees `gas: undefined` in the `transactionConfig` object. Here thrown [here](https://github.com/UMAprotocol/protocol/blob/master/packages/financial-templates-lib/src/logger/SlackTransport.js#L78)